src/test/resources/com/puppycrawl/tools/checkstyle/checks/blocks/rightcurly/InputRightCurlyTestAloneOrSingleline.java

```diff
@@ -244,4 +244,9 @@ public void codeAfterLastRightCurly() {
 
     interface Interface2
     { int i = 1; public void meth1(); }
+
+    private void foo28() {
+        if (true) { return; } else { } if (false) { // 2 violations
```

Comment:
This mutation kills https://github.com/checkstyle/checkstyle/pull/12618#discussion_r1066151940 .
It needs to be funky like this since it checks line positions.

Reply:
Good find :)